src/ui/library/Popover.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {useTheme} from '@ui/library';
```

Comment:
NIT:
Can be imported directly from `react-native-paper`

Reply:
I get `'useTheme' import from 'react-native-paper' is restricted. Please import useTheme from @ui/library instead  no-restricted-imports` lint error If i use from react-native-paper.

Reply:
Yes, this is a custom rule added by us to make sure we use the hook from the correct place. If the hook is imported from `@ui/library` this is actually coming from `react-native-paper`. But in this case we know what we are doing. That is to make the ui-library not depend on any configuration of the main project.

Let's say we change the tsconfig of the main project and `@ui/library` path is not valid anymore. We have to change here as well. That's why in this case you can disable the lint rule using `// eslint-disable-next-line no-restricted-imports`